lib/src/broadcast/mod.rs

```diff
@@ -0,0 +1,179 @@
+use futures_util::{SinkExt, StreamExt};
+use get_if_addrs::get_if_addrs;
+use log::{error, info};
```

Comment:
Can you remove this extra `.DS_store` mac files or maybe add this in `.gitignore`

Reply:
Done